plugins/module_utils/postgres.py

```diff
@@ -47,6 +51,24 @@ def postgres_common_argument_spec():
         connect_params=dict(default={}, type='dict'),
     )
 
+    # Setting the value for the port, if there is a PGPORT
+    if env_vars.get("PGPORT"):
+        dict_options.update(
+            port={
+                'type': 'int',
+                'default': int(env_vars.get("PGPORT")),
+                'aliases': ['login_port']
+            }
+        )
```

Comment:
```suggestion
        dict_options['port']['default'] = int(env_vars.get("PGPORT"))
```
wouldn't it look more readable?
or we could try to set it in dict() definition directly, something like:
```
port=dict(type='int', default=5432 if not env_vars.get("PGPORT") else int(env_vars.get("PGPORT")), aliases=['login_port']),
```
same for PGUSER

Just suggestions to avoid extra repetitive initializations + for better readability